### PR TITLE
Direct call fixes for JITServer on Power

### PR DESCRIPTION
**Pull request by dchopra001**
This is a double delivery of #7384

### CODE REVIEW

**Comment by dchopra001** [comment]
@mpirvu Can I get a review please?

File: runtime/compiler/p/codegen/J9PPCInstruction.cpp [hunk 1]
```diff
@@ -135,10 +136,20 @@ uint8_t *TR::PPCDepImmSymInstruction::generateBinaryEncoding()
             }
          }
 
-      if (cg()->comp()->compileRelocatableCode() && label == NULL)
+      if ((cg()->comp()->compileRelocatableCode() || cg()->comp()->isOutOfProcessCompilation()) &&
```
Comment (mpirvu) [inline runtime/compiler/p/codegen/J9PPCInstruction.cpp]: Unfortunetaly, in master we don't have `comp()->isOutOfProcessCompilation()` by default so we have to protected it with #ifdef JITSERVER_SUPPORT

Comment (dchopra001) [reply runtime/compiler/p/codegen/J9PPCInstruction.cpp]: Okay. I'll add the ifdef here and in other related PRs.

**Comment by dchopra001** [comment]
@mpirvu Ready for another review.

**Comment by mpirvu** [comment]
Jenkins test sanity plinux,aix jdk8,jdk11

**Comment by mpirvu** [comment]
Testing passes, hence merging.

**Pull request merged**